dbms/src/Flash/Coprocessor/RuntimeFilterMgr.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//
+// Created by elsa on 2023/2/8.
+//
+
+#pragma once
+
+#include <DataStreams/RuntimeFilter.h>
+
+#include <unordered_map>
+#include <vector>
+
+namespace DB {
+class RuntimeFilter;
+
+using RuntimeFilterPtr = std::shared_ptr<RuntimeFilter>;
+using RuntimeFilteList = std::vector<RuntimeFilterPtr>;
+extern RuntimeFilteList dummy_runtime_filter_list;
```

Comment:
Use std::optional?

Reply:
Doesn't seem to work? Because it is needed in Join.h